### PR TITLE
Generate Secret Word

### PR DESCRIPTION
**Pull request by kelseyroy**
Adds
- Answer with ReadWordsFile and GetRandomWord methods
- AnswerUnitTests 
- Data folders with 5_letter_words txt files in Wordle.Domain and Wordle.Test

### CODE REVIEW

File: Wordle.Domain/Answer.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+namespace Wordle.Domain;
+
+public class Answer
+{
+    public string GetRandomWord(string filePath)
+    {
+        var wordsArr = ReadWordsFile(filePath);
+        var i = new Random().Next(wordsArr.Length);
```
Comment (Aoxian) [inline Wordle.Domain/Answer.cs]: "Arr" is shorthand and exposes implementation.  This variable could jus be call `words`.

File: Wordle.Tests/AnswerUnitTests.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Wordle.Tests;
+
+public class AnswerUnitTests
+{
+    Answer answer = new Answer();
+    private static string relativePath = "../../../Data/5_letter_words_test.txt";
```
Comment (Aoxian) [inline Wordle.Tests/AnswerUnitTests.cs:12]: I like that you kept the test file with the tests. 👍

File: Wordle.Tests/AnswerUnitTests.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            "MODEL",
+            "AWARD",
+            "HOTLY",
+            "NICHE",
+            "JOUST",
+            "ARBOR",
+            "EERIE",
+            "CARAT",
+            "ROUGH"
+        };
```
Comment (Aoxian) [inline Wordle.Tests/AnswerUnitTests.cs]: could this be pulled out into a class variable? If the test file contents changes and the test grow then these word list would all have to change.

**Review by Aoxian — APPROVED** [review]
Left some comments, but overall this one looks good.

File: Wordle.Tests/AnswerUnitTests.cs (hunk 3 above)
Comment (ghost) [reply Wordle.Tests/AnswerUnitTests.cs]: maybe think about creating a words repository for different sources of words (future PR)

**Review by ghost — APPROVED** [review]

**Pull request merged**